src/main/scripts/jbang

```diff
@@ -9,6 +9,19 @@
 # The Java version to install when it's not installed on the system yet
 javaVersion=${JBANG_DEFAULT_JAVA_VERSION:-17}
 
+# Check if the script is running on Git Bash (MSYS environment)
+if [[ "$OSTYPE" == "msys" ]]; then
```

Comment:
I think you should move this down, below the place where we already do OS detection (eg around line 218) and check for `$os == windows` instead of using $OSTYPE.

Reply:
I would like to keep the code as it is. Hopefully we can remove this functionalty in the future. For that reason I would like to keep it separate from the rest of the code. This particular if statement (OSYTPE) also executes extremely quickly, and when it is not running in Git-Bash there will be no performance impact.